python/pyproject.toml

```
@@ -13,16 +13,53 @@
 # limitations under the License.
 
 [build-system]
-
+build-backend = "setuptools.build_meta:__legacy__"
 requires = [
     "wheel",
     "setuptools",
     "cython>=0.29,<0.30",
     "scikit-build>=0.13.1",
     "cmake>=3.23.1",
     "ninja",
-    "cuda-python>=11.7.1,<12.0"
+    "cuda-python>=11.7.1,<12.0",
+    "tomli  ; python_version < '3.11'",
```

This whitespace isn't necessary to my knowledge.
```suggestion
    "tomli; python_version < '3.11'",
```